src/app/move-popup/flavor-objective.scss

```diff
@@ -0,0 +1,9 @@
+.flavor-objective {
+  display: flex;
```

Comment:
It's best to explicitly state flex-direction here, and to set the `span` to be `flex: 1`.